boringtun/src/noise/timers.rs

```diff
@@ -6,14 +6,20 @@ use crate::noise::{Tunn, TunnResult};
 use std::mem;
 use std::ops::{Index, IndexMut};
 
-use std::time::{Duration, Instant};
+use std::time::Duration;
+
+#[cfg(test)]
+use mock_instant::Instant;
+
+#[cfg(not(test))]
+use std::time::Instant;
 
 // Some constants, represent time in seconds
 // https://www.wireguard.com/papers/wireguard.pdf#page=14
-const REKEY_AFTER_TIME: Duration = Duration::from_secs(120);
+pub const REKEY_AFTER_TIME: Duration = Duration::from_secs(120);
```

Comment:
We shouldn't expose any more internals at the lib crate level, especially just for tests. Try `pub(crate)` or `pub(super)`?

Reply:
Good call, done!